docker/README.md

```diff
@@ -2,85 +2,69 @@
 
 Docker compose environment to try out tiered storage plugins.
 
-> NOTE: Consider this playground environments unstable until future releases.
+The project uses a Docker image built from Aiven Kafka fork (e.g. https://github.com/aiven/kafka/tree/3.3-2022-10-06-tiered-storage) and https://github.com/aiven/tiered-storage-for-apache-kafka.
 
-## Structure
-
-- A [docker image](./kafka/Dockerfile) is built from a Kafka repository branch
-- Each plugin will have a Docker compose, e.g. [S3](./s3/compose.yml), to start an environment with Kafka, Zookeeper, plugin setup, and dependencies.
-
-## How to use
+## Requirements
 
-### Pre-requisites
+- Docker Compose
+- `make`
+- `kcat`
 
-Build plugin before starting Docker compose:
+## Structure
 
-#### Build plugin libraries
+- S3:
+  - s3/compose.yml: Expects environment variables to connect to AWS S3.
+  - s3-minio/compose.yml: Starts a local MinIO server to connect via S3 API.
+- Local:
+  - filesystem/compose.yml: Creates a local temporal directory with Tiered Storage plugin.
+  - test/compose.yml: Uses local directory for testing framework.
 
-On root directory:
+## How to use
 
-```shell
-./gradlew clean installDist
-```
+### Preparation
 
-#### Using Encryption mechanisms
+#### Connection to S3
 
-Generate RSA key pair:
+Set AWS credentials on `.env` file on `./s3` directory:
 
-```shell
-make rsa-keys
+```properties .env
+AWS_S3_BUCKET=test-kafka-ts
+AWS_REGION=us-west-1
+AWS_ACCESS_KEY_ID=...
+AWS_SECRET_ACCESS_KEY=...
 ```
 
-and set paths on `compose.yml` file:
+and use variables on `compose.yml`:
 
 ```yaml
   kafka:
     # ...
-    volumes:
-      # ...
-      - ./private.pem:/kafka/plugins/private.pem
-      - ./public.pem:/kafka/plugins/public.pem
-    command:
-      - kafka-server-start.sh
-      - config/server.properties
-      # ...
-      - --override
-      - rsm.config.s3.public_key_pem=/kafka/plugins/public.pem
-      - --override
-      - rsm.config.s3.private_key_pem=/kafka/plugins/private.pem
-      # ...
+    environment:
+      KAFKA_RSM_CONFIG_STORAGE_S3_BUCKET_NAME: ${AWS_S3_BUCKET}
+      KAFKA_RSM_CONFIG_STORAGE_S3_REGION: ${AWS_REGION}
+      KAFKA_RSM_CONFIG_STORAGE_AWS_ACCESS_KEY_ID: ${AWS_ACCESS_KEY_ID}
+      KAFKA_RSM_CONFIG_STORAGE_AWS_SECRET_ACCESS_KEY: ${AWS_SECRET_ACCESS_KEY}
```

Comment:
Since this env variables are already there, maybe this paragraph is redundant

Reply:
Ah: `KAFKA_RSM_CONFIG_STORAGE_AWS_ACCESS_KEY_ID` and `KAFKA_RSM_CONFIG_STORAGE_AWS_SECRET_ACCESS_KEY` are not in the `s3/compose.yml`. Maybe let's add them there and remove this paragraph from the README?